### PR TITLE
minor bar tweaks

### PR DESCRIPTION
**Pull request by ethn**
minor bar tweaks

### CODE REVIEW

File: card/mod/bootstrap/lib/stylesheets/style_bootstrap_cards.scss [hunk 1]
```diff
@@ -380,9 +380,30 @@ body.mceContentBody {
...
+  .media {
+    width: 100%
+  }
+  .media-left {
+    max-width: 25%
+  }
+  .bar-right {
+    word-break: break-word;
+  }
+}
```
Comment (ethn) [inline card/mod/bootstrap/lib/stylesheets/style_bootstrap_cards.scss]: this makes bars with media more responsive

File: card/mod/item/set/all/bar.rb [hunk 2]
```diff
@@ -51,23 +52,17 @@ def class_up_cols classes, cols, context=nil
 
   view :bar_left do
     class_up "card-title", "mb-0"
+    bar_title
+  end
+
+  def bar_title
```
Comment (ethn) [inline card/mod/item/set/all/bar.rb]: better yet, could make this a view.

File: decko/spec/cypress/integration/edit/pointer.spec.coffee [hunk 3]
```diff
@@ -56,7 +56,7 @@ describe 'editing pointers', () ->
         .should("contain", "Joe Camel")
       cy.contains(/select\s+3\s+following/)
       cy.get("input._select-all").click()
-      cy.contains(/select\s+0\s+following/)
+      # cy.contains(/select\s+0\s+following/)
```
Comment (ethn) [inline decko/spec/cypress/integration/edit/pointer.spec.coffee]: this appears very briefly and then disappears, so it fails sporadically if the timing is wrong.

Comment (xithan) [reply decko/spec/cypress/integration/edit/pointer.spec.coffee]: Why does it appear briefly? Shouldn't it remain 0 after I selected all?

Comment (ethn) [reply decko/spec/cypress/integration/edit/pointer.spec.coffee]: see the `count > 0` stuff in select_item.haml for more context.

I _think_ the rationale is that if there are no results, there is no need for a checkbox.  Another way to go would be to leave the checkbox there the whole time but disable if if there are 0 items to select...

**Pull request merged**